include/grpc++/impl/codegen/completion_queue.h

```diff
@@ -31,8 +31,18 @@
  *
  */
 
-/// A completion queue implements a concurrent producer-consumer queue, with two
-/// main methods, \a Next and \a AsyncNext.
+/// A completion queue implements a concurrent producer-consumer queue, with
+/// two main API-exposed methods: \a Next and \a AsyncNext. These
+/// methods are the essential component of the gRPC C++ asynchronous API.
+/// There is also a Shutdown method to indicate that a given completion queue
```

Comment:
Actually, we should decide whether we want to always use \a in front of function/method names. Right now we do it inconsistently (eg, the mentions of Read and Write wrt thread safety).